### DISCUSSION TITLE
How to make a SPAN an ENDPOINT

### DISCUSSION THREAD

**Discussion by 807801002** [apache/skywalking#7616] [Q&A] [answered]
I want to get the sla、cpm、avg response time of a method, I can make it be an SPAN by @Trace comment,but how to make it a ENDPOINT to show the sla、cpm、avg delay at dashboard？

**Comment by wu-sheng** [comment, marked as answer]
That is called `logic endpoint` in SkyWalking.

Check the doc, https://skywalking.apache.org/docs/skywalking-java/latest/en/setup/service-agent/java-agent/logic-endpoint/#the-logic-endpoint
